openlibrary/macros/QueryCarousel.html

```diff
@@ -1,4 +1,4 @@
-$def with(query, title=None, sort='new', key='', limit=20, search=False, has_fulltext_only=True, url=None, layout='carousel', use_cache=True)
+$def with(query, title=None, sort='new', key='', limit=20, search=False, has_fulltext_only=True, url=None)
```

Comment:
Changes to this file should be reverted.  Instead, make changes in [RawQueryCarousel.html](https://github.com/internetarchive/openlibrary/blob/master/openlibrary/macros/RawQueryCarousel.html).